virtualizarr/manifests/manifest.py

```diff
@@ -285,7 +299,7 @@ def to_zarr_json(self, filepath: str) -> None:
     @classmethod
     def _from_kerchunk_chunk_dict(cls, kerchunk_chunk_dict) -> "ChunkManifest":
         chunkentries = {
-            cast(ChunkKey, k): ChunkEntry.from_kerchunk(v).dict()
+            cast(ChunkKey, k): ChunkEntry.from_kerchunk(tuple(v)).dict()
```

Comment:
I _think_ if you don't add this `tuple()` call then my suggestion of using `isinstance(..., str)` above will work.

Reply:
Ya, with my last commit the validation takes care of this.